fuel/app/classes/materia/fuel/core/cookie.php

```diff
@@ -41,7 +41,8 @@ public static function set($name, $value, $expiration = null, $path = null, $dom
 		// is_null($secure) and $secure = static::$config['secure'];
 		is_null($http_only) and $http_only = static::$config['http_only'];
 		//static::$config is protected - can't get it in an extended class, hack workaround here
-		is_null($same_site) and $same_site = 'None';
+		// FuelPHP casts $same_site as a string, which converts null into an empty string. Just using is_null is insufficient.
+		if ( is_null($same_site) || ! $same_site) $same_site = 'None';
```

Comment:
In hindsight, `is_null($same_site)` should also be covered with `! $same_site` so we may not even need both of them.

Reply:
Removed `is_null` due to its redundancy.